sql/pg_net.sql

```diff
@@ -353,5 +353,14 @@ begin
 end;
 $$;
 
+create or replace function net.worker_restart() returns bool as $$
+  select pg_reload_conf();
+  select pg_terminate_backend(pid)
+  from pg_stat_activity
+  where backend_type ilike '%pg_net%';
+$$
+security definer
+language sql;
+
```

Comment:
We also need to bump the version and add a migration

Reply:
Yes, but that can be done on another PR that bumps the version. I plan on adding a few more fixes before that.